index.js

```diff
@@ -320,7 +320,7 @@ let Fixtures = {
             let request = {
                 url: '/' + VERSION + '/' + model.module,
                 method: 'POST',
-                data: model.attributes || {},
+                data: _.cloneDeep(model.attributes) || {},
```

Comment:
This portion addresses the problem discussed [here](https://github.com/sugarcrm/thorn/pull/134#discussion_r102377004).

TL;DR

If someone declares a fixture and relies on it after calling `Fixtures.create`, if that fixture has required fields that are not supplied as attributes then the original fixture changes when we automatically populate those required attributes/fields.